user.rb

```diff
@@ -26,4 +26,15 @@ def save
       puts "User Created Successfully"
     end
   end
+
+  def self.find_user(email)
```

Comment:
`find_by_email` makes much more sense, we are already inside user class, no neeed to add `user` in a method name